flink-ml-lib/src/test/java/org/apache/flink/ml/feature/UnivariateFeatureSelectorTest.java

```diff
@@ -0,0 +1,771 @@
+                                    0.79686718,
+                                    0.55776083,
+                                    0.96583653,
+                                    0.1471569,
+                                    0.029647,
+                                    0.59389349)),
+                    Row.of(
+                            0.57739736,
+                            Vectors.dense(
+                                    0.1140657,
+                                    0.95080985,
+                                    0.96583653,
+                                    0.19361869,
+                                    0.45781165,
+                                    0.92040257)),
+                    Row.of(
+                            0.53877145,
+                            Vectors.dense(
+                                    0.87906916,
+                                    0.25261576,
+                                    0.34800879,
+                                    0.18258873,
+                                    0.90179605,
+                                    0.70652816)));
+
+    @Before
+    public void before() {
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        env.getConfig().enableObjectReuse();
+        tEnv = StreamTableEnvironment.create(env);
+
+        selectorWithChiSqTest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("categorical")
+                        .setLabelType("categorical");
+        selectorWithANOVATest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("continuous")
+                        .setLabelType("categorical");
+        selectorWithFValueTest =
+                new UnivariateFeatureSelector()
+                        .setFeatureType("continuous")
+                        .setLabelType("continuous");
+        inputChiSqTable =
+                tEnv.fromDataStream(env.fromCollection(INPUT_CHISQ_DATA)).as("label", "features");
+        inputANOVATable =
+                tEnv.fromDataStream(env.fromCollection(INPUT_ANOVA_DATA)).as("label", "features");
+        inputFValueTable =
+                tEnv.fromDataStream(env.fromCollection(INPUT_FVALUE_DATA)).as("label", "features");
+    }
+
+    private void transformAndVerify(
+            UnivariateFeatureSelector selector, Table table, int... expectedIndices)
+            throws Exception {
+        UnivariateFeatureSelectorModel model = selector.fit(table);
+        Table output = model.transform(table)[0];
+        verify(output, expectedIndices);
+    }
+
+    private void verify(Table table, int... expectedIndices) throws Exception {
```

Comment:
It might be better to modify this method's signature into follows.
```java
private void verifyOutputResult(Table outputTable, int... expectedIndices) throws Exception
```